src/styling/about.css

```diff
@@ -28,4 +32,11 @@ ul {
 .container {
   margin-top: 100px;
   margin-bottom: 100px;
+}
+
+.socialLogo {
+  height: 30px;
+  width: 30px;
```

Comment:
Could you remove this line of styling? It's making the LinkedIn logo look squished and the images look okay without it.